templates/emails/new-mention.php

```diff
@@ -46,19 +48,21 @@
 	?>
 </div>
 
+<?php if ( site_supports_blocks() && ! \is_plugin_active( 'classic-editor/classic-editor.php' ) ) : ?>
 <p>
-	<a class="button" href="<?php echo esc_url( admin_url( 'post-new.php?in_reply_to=' . $args['activity']['object']['id'] ) ); ?>">
-		<?php esc_html_e( 'Reply to the post', 'activitypub' ); ?>
+	<a class="button" href="<?php echo \esc_url( \admin_url( 'post-new.php?in_reply_to=' . $args['activity']['object']['id'] ) ); ?>">
+		<?php \esc_html_e( 'Reply to the post', 'activitypub' ); ?>
 	</a>
 </p>
+<?php endif; ?>
```

Comment:
What's an alternative action that we could suggest here? View the post?

Reply:
I think removing it is fine... The classic editor is the exception, so we should simply care about not producing any errors.